test/issue1551-var-escaping/dub.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "issue1551-var-escaping",
+    "preGenerateCommands": [
+        "echo $${DUB_PACKAGE_DIR}",
+        "echo $$DUB_PACKAGE_DIR"
```

Comment:
Not a real test.

Reply:
Good point, that looks like a leftover. But this test was written before the unittests were put in place.